### PR TITLE
Account for different upstream/downstream ipxe filenames

### PR DESCRIPTION
**Pull request by juliakreger**

### CODE REVIEW

**Comment by derekhiggins** [comment]
I don't see prepare-ipxe.sh, did you forget to add it? 
I'm also wondering what was wrong with the downstream only version (changed here https://github.com/openshift/ironic-image/pull/9), I didn't think openshift ci would be looking at this .ocp file)

File: prepare-ipxe.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+mkdir /tftpboot
+cp /usr/share/ipxe/undionly.kpxe /tftpboot/
+if [ -f "/usr/share/ipxe/ipxe.efi" ]; then
+    cp /usr/share/ipxe/ipxe.efi /tftpboot/ipxe.efi
+elif [ -f "/usr/share/ipxe/ipxe-x86_64.efi" ]; then
+    cp  /usr/share/ipxe/ipxe-x86_64.efi /tftpboot/ipxe.efi
+else
+    echo "Fatal Error - Failed to find ipxe binary"
+    exit 1
+fi /usr/share/ipxe/ipxe.efi /tftpboot/
```
Comment (derekhiggins) [inline prepare-ipxe.sh]: typo here
```
/tmp/t: line 11: syntax error near unexpected token `/usr/share/ipxe/ipxe.efi'
/tmp/t: line 11: `fi /usr/share/ipxe/ipxe.efi /tftpboot/'

```

Comment (derekhiggins) [reply prepare-ipxe.sh]: Does there need to be two commits here?

**Comment by derekhiggins** [comment]
Does there need to be two commits here?

**Comment by elfosardo** [comment]
@dtantsur we should, actually we should keep upstream and downstream as close as possible
until centos8 is not here though, it looks hard, unless we could use ubi8

**Comment by dtantsur** [comment]
If we can confirm this code works with RHEL 8, we know it will work for CentOS 8, no?

**Comment by derekhiggins** [comment]
> Does there need to be two commits here?

Now there is 3 commits, can we squash them into one?

**Comment by juliakreger** [comment]
@dtantsur only once the downstream image sync occurs, and once merged it will be an email tomorrow.

**Comment by juliakreger** [comment]
@derekhiggins Can we just squash with merge buttin in the github ui?

**Comment by juliakreger** [comment]
@elfosardo We are using ubi8. The downstream team has no idea why this is happening :\

**Comment by juliakreger** [comment]
FYI, the OpenShift nightly build failed because of this, so it would be good to move this forward ASAP as they are wanting to re-trigger the nightly builds.

File: Dockerfile.ocp [hunk 2]
```diff
@@ -6,9 +6,9 @@ RUN yum update -y && \
         mariadb-server python3-PyMySQL python2-chardet ipxe-roms-qemu && \
     yum clean all
 
-RUN mkdir /tftpboot && \
-    cp /usr/share/ipxe/undionly.kpxe /tftpboot/ && \
-    cp /usr/share/ipxe/ipxe.efi /tftpboot/ipxe.efi
+copy ./prepare-ipxe.sh /tmp
```
Comment (dtantsur) [inline Dockerfile.ocp]: nit: please update it to `COPY` when implementing upstream

File: prepare-ipxe.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -x
+mkdir /tftpboot
```
Comment (dtantsur) [inline prepare-ipxe.sh]: nit: `mkdir -p`

**Comment by derekhiggins** [comment]
> @derekhiggins Can we just squash with merge buttin in the github ui?

yup, have never that before but works for me

**Pull request merged**